### PR TITLE
Updating swagger for data flow api

### PR DESCRIPTION
**Pull request by Nexengineer**

### CODE REVIEW

File: src/In.ProjectEKA.HipService/wwwroot/swagger.yaml [hunk 1]
```diff
@@ -221,6 +222,69 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+
+  /health-information:
```
Comment (Apoorva-GA) [inline src/In.ProjectEKA.HipService/wwwroot/swagger.yaml]: Can we have it same as CM `/health-information/request`

File: src/In.ProjectEKA.HipService/wwwroot/swagger.yaml [hunk 2]
```diff
@@ -221,6 +222,69 @@ paths:
...
+            **Causes:**
+              * Consent Artefact id not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401':
```
Comment (Apoorva-GA) [inline src/In.ProjectEKA.HipService/wwwroot/swagger.yaml]: When will the request be Unauthorized? Please add it as part of description

File: src/In.ProjectEKA.HipService/wwwroot/swagger.yaml [hunk 3]
```diff
@@ -433,4 +499,56 @@ components:
...
+                $ref: '#/components/schemas/CareContextRepresentation'
+
+    HealthInformationRequest:
+      type: object
+      required:
+        - transactionId
+        - consent
+        - callBackUrl
+        - keyMaterial
+        - hiDataRange
```
Comment (Apoorva-GA) [inline src/In.ProjectEKA.HipService/wwwroot/swagger.yaml]: Can be optional

File: src/In.ProjectEKA.HipService/wwwroot/swagger.yaml [hunk 4]
```diff
@@ -221,6 +222,69 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+
+  /health-information:
+    post:
```
Comment (Apoorva-GA) [inline src/In.ProjectEKA.HipService/wwwroot/swagger.yaml]: Don't we have to authorize the Api?

Comment (Nexengineer) [reply src/In.ProjectEKA.HipService/wwwroot/swagger.yaml]: There is a consentmanager id in middleware which will be authorized by JWT or Mutual authentication

File: src/In.ProjectEKA.HipService/wwwroot/swagger.yaml (hunk 1 above)
Comment (Nexengineer) [reply src/In.ProjectEKA.HipService/wwwroot/swagger.yaml]: sure

**Pull request merged**